src/utilities/util_alter.c

```diff
@@ -8,13 +8,21 @@
 
 #include "util.h"
 
+/*
+ * usage --
```

Comment:
I added a TODO as part of the boilerplate comment. What do you guys think of this?

Reply:
I think we should have a TODO there. Or, alternatively, a FIXME.